src/platform/forms-system/src/js/helpers.js

```diff
@@ -145,14 +146,15 @@ export function formatReviewDate(dateString, monthYear = false) {
 
   return undefined;
 }
+
 export function parseISODate(dateString) {
   if (typeof dateString === 'string') {
     const [year, month, day] = dateString.split('-', 3);
 
     return {
       month: month === 'XX' ? '' : Number(month).toString(),
       day: day === 'XX' ? '' : Number(day).toString(),
-      year: year === 'XXXX' ? '' : year,
+      year: year === 'XXXX' ? '' : Number(year).toString(),
```

Comment:
@jbalboni do you foresee any issues with this? What we're trying to do here is remove the leading zeros by converting this to a number and back to a string. Do you have any historical context as to why we hadn't done this yet?

Reply:
I should mention that this isn't a full fix, though it improves things _visually_ by removing the leading 0s. The form value still contains leading 0s. Also, I only looked into it for a few hours, but an otherwise valid year with leading 0s (e.g. 0001985) seemed to prevent me from advancing to the next form page, though no errors were displayed. That issue seemed to occur with and without my change.

Reply:
I can't really think of anything, but it's been quite a while since I looked at this. I don't remember why we were treating the year differently.

Reply:
Sounds good. Thanks for checking in, @jbalboni! :raised_hands: 

@nathanielbuck You can merge this whenever you're ready. This looks good to me. :shipit: